@here/harp-mapview/lib/text/TextElementsRenderer.ts

```diff
@@ -1739,11 +1750,6 @@ export class TextElementsRenderer {
             if (iconInvisible) {
                 iconRenderState.reset();
             }
-        } else if (renderIcon && poiInfo!.isValid !== false) {
-            // Ensure that text elements still loading icons get a chance to be rendered if
-            // there are no text element updates in the next frames.
-            this.m_forceNewLabelsPass = true;
-            this.m_viewUpdateCallback();
```

Comment:
what happens with icon images that are still loading?

Reply:
they were already covered by the loading state, which was not evaluated correctly due to this unfortunate flag in  MapView. Test are all passing and as far as I see everything still works. I ll keep an eye on it